backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -63,11 +65,27 @@ public void testCreateShop() throws Exception {
         assertEquals("Shop Phone Number ", PHONE_NUMBER, inMemoryVirtualQueueRepository.getShopMap().get(shopId).getPhoneNumber());
         assertEquals("Shop Type ", SHOP_TYPE, inMemoryVirtualQueueRepository.getShopMap().get(shopId).getShopType());
     }
-    
+
     @Test
     public void testGetWaitingTime() throws Exception {
         long waitingTime = inMemoryVirtualQueueRepository.getWaitingTime();
-        
+
         assertEquals("Waiting Time per customer is", WAITING_TIME_MINS, waitingTime);
     }
+
+    @Test
+    public void testGetNewToken_success() {
+        CreateShopRequest shop = new CreateShopRequest(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
+        UUID shopId = inMemoryVirtualQueueRepository.createShop(shop);
+        Token newToken = inMemoryVirtualQueueRepository.getNewToken(shopId);
+        
+        UUID tokenId = newToken.getTokenId();
+        int actualTokenMapSize = inMemoryVirtualQueueRepository.getTokenMap().size();
```

Comment:
replace actual with expected. Here and in other variable names

Reply:
Done.